onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -0,0 +1,1047 @@
+  no_dimension_supported.push_back({V_2021_2,{"All"},"Where"});
+  no_dimension_supported.push_back({V_2021_2,{"All"},"Range"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"ArgMin"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Max"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Add"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Less"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Greater"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Clip"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Resize"});
+  no_dimension_supported.push_back({V_2021_2,{"Myriad"},"Equal"});
+
+
+  subgraph_supported.push_back({V_2020_4,{"All"},"Mul"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Transpose"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Unsqueeze"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Cast"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Concat"});
+  subgraph_supported.push_back({V_2020_4,{"All"},"Gather"});
+  subgraph_supported.push_back({V_2020_4,{"Myriad"},"Div"});
+  subgraph_supported.push_back({V_2020_4,{"Myriad"},"Sub"});
+  subgraph_supported.push_back({V_2021_1,{"CPU"},"Identity"});
+  subgraph_supported.push_back({V_2021_1,{"CPU"},"Div"});
+  subgraph_supported.push_back({V_2021_1,{"CPU"},"Sub"});
+    
+  
+  //populate unsupportedmode_t
+  {
+    unsupportedopmode_t obj = {{V_2020_4,V_2021_1,V_2021_2},  
+     [this](const Node* node, const Provider_InitializedTensorSet&) {
+      for (size_t i = 0; i < node->InputDefs().size(); i++) {
+        if (node->InputDefs()[i]->TypeAsProto()->tensor_type().elem_type() != ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT)
+          return true;
+      }
+      return false;
+    }
+    };
+    _confirmation_map.insert({"Abs", obj});
+  }
+  {
+     unsupportedopmode_t obj = {{V_2020_4},
+      [this](const Node* node, const Provider_InitializedTensorSet&) {
+        auto& attributes = node->GetAttributes();
+        auto last_index_arg = attributes.count("select_last_index") > 0 ? attributes.at("select_last_index").i() : 0;
+        if (last_index_arg != 0)
+          return true;
+        if (node->InputDefs()[0]->TypeAsProto()->tensor_type().elem_type() != ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT)
+          return true;
+        return false;  
+      }
+    };
+    _confirmation_map.insert({"ArgMax", obj});
+    _confirmation_map.insert({"ArgMin", obj});
+  }
+  {
+    unsupportedopmode_t obj = {{V_2020_4},
+      [this](const Node* node, const Provider_InitializedTensorSet&) {
+        const auto& attributes = node->GetAttributes();
+        auto ceil_attr = attributes.find("ceil_mode");
+        if (ceil_attr != attributes.end() && ceil_attr->second().i() != 0) return true;
+        return check_if_dimension_unsupported(node);
+      }
+    };
+    _confirmation_map.insert({"AveragePool", obj});
```

Comment:
Please change confirmation_map name to something else

Reply:
Resolved